pro/src/ui-kit/form/shared/BaseCheckbox/BaseCheckbox.tsx

```diff
@@ -33,6 +33,7 @@ const BaseCheckbox = ({
         className={cn(styles['base-checkbox-input'], {
           [styles['has-error']]: hasError,
         })}
+        data-testid={'checkbox'}
```

Comment:
```suggestion
        data-testid={'base-checkbox'}
```

Mais tu ne devrait plus en avoir besoin si tu change le selecteur du test avec le isDuo.

Reply:
jamais réussi avec le get by label je me suis demandé s'il y avait pas un truc bizarre dans le parent